ui/src/app/components/help-sidebar.tsx

```diff
@@ -235,6 +237,13 @@ export const HelpSidebar = fp.flow(
 
     setActiveIcon(activeIcon: SidebarIconId) {
```

Comment:
`runtimeConfPanelInitialState` is only used by AppsPanel to force the Runtime Config Panel into a specific state.

This function is called in many other contexts where that is not happening, and we do not want `runtimeConfPanelInitialState` to be set.

See just below here, for `openRuntimeConfigWithState()` which is the only case where we DO want the value to be set.

This can probably be improved.  ideas?